gradle/versions.gradle

```diff
@@ -84,7 +84,8 @@ dependencyManagement {
     }
 
     dependency 'org.mockito:mockito-core:3.1.0'
-    
+    dependency 'org.mockito:mockito-junit-jupiter:3.1.0'
```

Comment:
We can avoid adding this dependency if we stay consistent with the `Mockito.mock` approach to creating mocks right?

Reply:
reverted the dependency.